### PR TITLE
OC-8254:Scheduled CRF import creates duplicate event CRF if a differe…

### PR DESCRIPTION
**Pull request by Taoli2018**
…nt CRF version already has been started for that event.

### CODE REVIEW

File: web/src/main/java/org/akaza/openclinica/web/job/ImportSpringJob.java [hunk 1]
```diff
@@ -624,11 +624,29 @@ private String generateMsg(String msg, String contactEmail) {
...
+                             *  now  it's time to update database for the migrated CRF version
+                             */
+                            int currentCRFVersionId = eventCrfBean.getCRFVersionId();
+              			    int newCRFVersionId =  displayItemBean.getMetadata().getCrfVersionId();
+              			    if(currentCRFVersionId != newCRFVersionId && eventCrfBeanIdProcessed != eventCrfBeanId) {
+              			    	eventCrfDao.updateCRFVersionID(eventCrfBeanId, newCRFVersionId, ub.getId());
+              			    	eventCrfBeanIdProcessed = eventCrfBeanId;
+
+              			    	eventCrfBean.setCRFVersionId(newCRFVersionId);
+              			    }
```
Comment (svadla-oc) [inline web/src/main/java/org/akaza/openclinica/web/job/ImportSpringJob.java:648]: fix indentation

**Review by svadla-oc — CHANGES_REQUESTED** [review]

**Comment by svadla-oc** [comment]
This should be created for 3.17.x branch

**Comment by Taoli2018** [comment]
please also see new PR in 3.17x
https://github.com/OpenClinica/OpenClinica/pull/3474

**Comment by svadla-oc** [comment]
Closing since this is included in #3474

**Pull request closed without merging**